docs/project/tokenomics.mdx

```diff
@@ -39,22 +39,24 @@ The following sections describe how the smart contract issues new tokens, and ho
 
 | Distribution               | Percentage  | Amount         |
 | ---------------------------| ----------- | -------------- |
-| **Station Rewards**      | 52%         | 52,000,000 WXM |
+| **Daily Station Rewards**      | 52%         | 52,000,000 WXM |

```

Comment:
Let's keep "Station Rewards" as mentioned in tokenomics on the website as well? By having "Daily" here someone might think that we issue 52M daily.

Reply:
ok